src/freva/cli/history.py

```diff
@@ -77,26 +82,42 @@ def __init__(
             action="store_true",
             default=False,
         )
+        self.parser.add_argument(
+            "--json",
+            "-j",
+            help=(
+                "Display a json representation of the output, this can be"
+                "useful if you want to build shell based pipelines and want"
+                "parse the output with help of `jq`."
+            ),
```

Comment:
```
                "Display a json representation of the output, this can be "
                "useful if you want to build shell based pipelines and want to "
                "parse the output with the help of `jq`."